CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # bedrock-quasar ChangeLog
 
+## 10.0.0 - 2024-xx-xx
+
+### Added
+- Add support for Quasar initialization options.
+
+### Changed
+- Update dependencies.
+- **BREAKING**: Remove bedrock override to use quasar UMD bundle.
+  - Any app using this package and `bedrock-webpack` is effected by this change
+    and may need to update imports due to differences in the UMD vs ESM quasar
+    bundles.
+  - Change `import Quasar from 'quasar';` to `import {Quasar} from 'quasar';`.
+  - Change `import {default as Quasar} from 'quasar';` to `import {Quasar} from
```

Comment:
There's some issue with backticks here.

Reply:
I think it's rendering HTML correctly?  Are newlines not supposed to be used in md backtick blocks?